play25-json-ops/src/main/scala/play/api/libs/json/ops/RecoverOps.scala

```diff
@@ -0,0 +1,111 @@
+package play.api.libs.json.ops
+
+import play.api.data.validation.ValidationError
+import play.api.libs.json._
+
+import scala.language.higherKinds
+import scala.reflect.ClassTag
+import scala.util.control.NonFatal
+
+trait RecoverOps[F[x] <: Reads[x], A] extends Any {
```

Comment:
Why the `extends Any`?  That's a no-op, AFAIK.

Reply:
It makes the trait a "universal" trait, so that it can be extended by value classes.